custom-prom.md

```diff
@@ -3,15 +3,28 @@ Prometheus Configuration Guide
 
 ## Bring your own Prometheus
 
-When integrating Kubecost with an existing Prometheus, we recommend first installing Kubecost with a bundled Prometheus ([instructions](http://kubecost.com/install)) as a dry run before integrating with an external Prometheus deployment. You can get in touch (support@kubecost.com) or via our [Slack community](https://join.slack.com/t/kubecost/shared_invite/zt-1dz4a0bb4-InvSsHr9SQsT_D5PBle2rw) for assistance.
+When integrating Kubecost with an existing Prometheus, we _strongly_ recommend installing Kubecost with a bundled Prometheus ([instructions](http://kubecost.com/install)).
 
-The Kubecost Prometheus deployment is used both as a source and store of metrics. It’s optimized to not interfere with other observability instrumentation and by default only contains metrics that are useful to the Kubecost product. This results in __70-90% fewer metrics__ than a Prometheus deployment using default settings.
+The Kubecost Prometheus deployment is optimized to not interfere with other observability instrumentation and by default only contains metrics that are useful to the Kubecost product. This results in __70-90% fewer metrics__ than a Prometheus deployment using default settings.
 
-For the best experience, we generally recommend teams use the bundled prometheus-server & grafana but reuse their existing kube-state-metrics and node-exporter deployments if they already exist. This setup allows for the easiest installation process, easiest ongoing maintenance, minimal duplication of metrics, and more flexible metric retention.
+Additonally, if multi-cluster metric aggregation is required, Kubecost provides a turnkey solution that is highly tuned and simple to support using the included Prometheus deployment.
 
 > **Note**: the Kubecost team provides best efforts support for free/community users when integrating with an existing Prometheus deployment.
 
+## Disable node-exporter and kube-state-metrics (recomended)
 
+If you have node-exporter and/or KSM running on your cluster, follow this step to disable the Kubecost included versions.
```

Comment:
We should consider adding a link to the KSM docs, specifically the duplicate metrics section. We don't want to recommend disabling KSM unless they are running V1 or V2 with the relevant helm flags set.